Formula/treefmt.rb

```diff
@@ -0,0 +1,20 @@
+class Treefmt < Formula
+  desc "One CLI to format the code tree"
+  homepage "https://github.com/numtide/treefmt"
+  url "https://github.com/numtide/treefmt/archive/refs/tags/v0.4.1.tar.gz"
+  sha256 "48554e1b030781c49c98c5882369a92e475d76fee0d5ce2d2f79966826447086"
+  license "MIT"
+
+  head "https://github.com/numtide/treefmt.git", branch: "master"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--path", ".", "--root", buildpath
+    bin.install buildpath / "bin" / "treefmt"
```

Comment:
```suggestion
    system "cargo", "install", *std_cargo_args
```
Does this not work?

Reply:
@carlocab Thanks, I hadn't seen `std_cargo_args` documented. It seems to work but these additional crates files are installed. I assume this is not desirable?

```
% brew list treefmt
/opt/homebrew/Cellar/treefmt/0.4.1/.crates.toml
/opt/homebrew/Cellar/treefmt/0.4.1/.crates2.json
/opt/homebrew/Cellar/treefmt/0.4.1/bin/treefm
```

Reply:
That's fine. You can remove them if you like, but I don't think removing them is worth the extra code. (Avoiding them isn't worth not using `std_cargo_args` either.)

Reply:
It doesn't bother me either way so have made accepted the change.